### DISCUSSION TITLE
Share with MCNP Forum

### DISCUSSION THREAD

**Discussion by SterlingButters** [idaholab/MontePy#356] [General]
Has this project been shared with the MNCP forum? I'm happy to do it if not

**Comment by MicahGale** [comment]
I intend to; there's been some hickups on our side.

**Reply by SterlingButters** [reply]
Sounds good, I won't mention it then

**Reply by MicahGale** [reply]
Check your email.

**Reply by SterlingButters** [reply]
Get ready for the repo to explode

**Reply by MicahGale** [reply]
Here's hoping.
